taf/updater/updater_pipeline.py

```diff
@@ -657,7 +657,13 @@ def _validate_last_validated_commit(self, last_validated_commit):
             if users_head_sha not in commits_since:
                 msg = f"Top commit of repository {self.state.users_auth_repo.name} {users_head_sha} and is not equal to or newer than last successful commit"
                 taf_logger.error(msg)
-                raise UpdateFailedError(msg)
+                if self.force:
+                    taf_logger.info(
+                        "Forcing update: setting last validated commit to the top commit of the local repository."
+                    )
+                    self.state.users_auth_repo.set_last_validated_commit(users_head_sha)
+                else:
+                    raise UpdateFailedError(msg)
```

Comment:
The problem here is that we don't really know if the commit is even valid. I don't think there is any other way to check that without checking if the remote repo contains it.